tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -141,38 +131,38 @@ KUBETEST2_ARGS=()
 KUBETEST2_ARGS+=("-v=2")
 KUBETEST2_ARGS+=("--cloud-provider=${CLOUD_PROVIDER}")
 KUBETEST2_ARGS+=("--cluster-name=${CLUSTER_NAME:-}")
-KUBETEST2_ARGS+=("--kops-binary-path=${KOPS_BIN}")
+KUBETEST2_ARGS+=("--kops-version-marker=https://storage.googleapis.com/k8s-staging-kops/kops/releases/markers/master/latest-ci.txt")
 KUBETEST2_ARGS+=("--admin-access=${ADMIN_ACCESS:-}")
 KUBETEST2_ARGS+=("--env=KOPS_FEATURE_FLAGS=${KOPS_FEATURE_FLAGS}")
 
+if [[ "${CLOUD_PROVIDER}" == "gce" ]]; then
+  KUBETEST2_ARGS+=("--boskos-resource-type=scalability-scale-project")
+  KUBETEST2_ARGS+=("--control-plane-instance-group-overrides=spec.rootVolume.type=pd-ssd")
+  KUBETEST2_ARGS+=("--max-nodes-to-dump=10")
+fi
+
 # More time for bigger clusters
 KUBETEST2_ARGS+=("--validation-wait=55m")
 KUBETEST2_ARGS+=("--validation-count=3")
-KUBETEST2_ARGS+=("--validation-interval=60s")
+KUBETEST2_ARGS+=("--validation-interval=70s")
```

Comment:
Why increase to 70s?

Reply:
I did 70 first and then forgot to fix it when I rebased it.

Reply:
could you remove this part? I don't have any other objections on the merge.